BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/DeleteAccount/DeleteAccountProcessorTests.swift

```diff
@@ -164,7 +164,7 @@ class DeleteAccountProcessorTests: BitwardenTestCase {
         XCTAssertTrue(authRepository.requestOtpCalled)
         coordinator.loadingOverlaysShown.removeAll()
 
-        var textField = try XCTUnwrap(alert.alertTextFields.first)
+        _ = try XCTUnwrap(alert.alertTextFields.first)
```

Comment:
🎨 I think we can remove this line since `setText` does this, sorry that was my bad.
```suggestion
```

Reply:
No problem, left it in case you were trying to enforce that there were at least one text field in the alert before acting on it. But I'll commit your suggestion and ask the re-review 😄